packages/build-tools/src/buildErrors/userErrorHandlers.ts

```diff
@@ -256,7 +256,7 @@ To resolve this issue, downgrade to an older Xcode version using the "image" fie
     createError: () =>
       new UserFacingError(
         errors.ErrorCode.UNKNOWN_FASTLANE_ERROR,
-        `Fastlane build failed with unknown error. See logs for the "Run fastlane" and "Xcode Logs" phases for more information.
+        `Fastlane build failed with unknown error. See logs for the "Run fastlane" (it contains filtered build logs) and "Xcode Logs" (it contains full build logs) phases for more information.
```

Comment:
```suggestion
        `The "Run fastlane" step failed with an unknown error. Refer to "Xcode Logs" below for additional, more detailed logs.
```

also, let's auto-expand the "Xcode Logs" phase whenever a build is failed and Xcode Logs are available

Reply:
This message will show up at the end of the build logs, but also in EAS CLI, so bellow will not make much sense

Reply:
@brentvatne @wkozyra95 @dsokal I have an idea of how we can improve it 🤔. Maybe we can use the changes I'm implementing when working on https://linear.app/expo/issue/ENG-7376/add-link-directly-to-log-build-phase-in-eas-cli-build-error-message and add error messages overrides in the EAS CLI (I need to add similar improvements either way as a part of this task)? 

This way we can have the error message `The "Run fastlane" step failed with an unknown error. Refer to the "Xcode Logs" below for additional, more detailed logs` on the website with the auto-opening of the `Xcode Logs` phase and message with a link to the build phase in the EAS CLI `The "Run fastlane" step failed with an unknown error. Refer to the ["Xcode Logs"](https://expo.dev/accounts/:id/projects/:id/build/:id#xcode-logs) phase for additional, more detailed logs`.

Or maybe we want to have the version with a link to the build phase in two places and display only server-side error messages both in the EAS CLI and on the website? What do you think?

Reply:
In the current version it might be tricky to create an error message with a link on the Turtle side because we need to know the project and account id and I'm not sure if we want to put this kind of information in Turtle 🤔, so I'm leaning towards the first approach. Although if I implement this part of the task: `A side note - these URLs are so long. I'd love to just have expo.dev/builds/id be our default.`  it wouldn't be a problem at all.

Reply:
Either way, I think it needs to be implemented as part of https://linear.app/expo/issue/ENG-7376/add-link-directly-to-log-build-phase-in-eas-cli-build-error-message, and the process of changing the error messages and making them use links.

Reply:
if you use https://github.com/expo/eas-cli/blob/main/packages/eas-cli/src/log.ts#L91 you can provide different text, than url you are directing to. If the terminal is not supported it will still show full URL

Reply:
Thanks! Currently, I'm leaning towards creating error overrides in the EAS CLI as the best approach to solve this issue. I will merge this PR with the error message suggested by Brent (website version) and add auto-expanding of the `Xcode Logs` build phase after error overrides are released so we already have a custom override of this message in the CLI in place.